README.md

```diff
@@ -106,6 +106,37 @@ In this case as parameter separator it's possible to use whatever character is n
 
 Having a route with multiple parameters may affect negatively the performance, so prefer single parameter approach whenever possible, especially on routes which are on the hot path of your application.
 
+<a name="match-order"></a>
+##### Match order
+The routes are matched in the following order:
+```
+static
+parametric
+wildcards
+parametric(regex)
+multi parametric(regex)
+```
+This means that if you register two routes, the first static and the second dynamic with a shared part of the path, the static route will always take precedence.
+For example:
+```js
+const assert = require('assert')
+const router = require('find-my-way')({
+  defaultRoute: (req, res) => {
+    assert(req.url === '/example/shared/nested/oops')
+  }
+})
+
+router.on('GET', '/example/shared/nested/test', (req, res, params) => {
+  assert.fail('We should not be here')
+})
+
+router.on('GET', '/example/:param/nested/test', (req, res, params) => {
```

Comment:
maybe this should be  `/example/:param/nested/oops`

Reply:
Why?
That example is just to explain the behaviour that is covered by [this](https://github.com/delvedor/find-my-way/blob/find-refactor/test/methods.test.js#L420-L439) test and explained few lines above.

Reply:
I find this example better explaining the concept:

```js
  findMyWay.lookup({ method: 'GET', url: '/example/shared/nested/oops' }, null)
```

Reply:
I agree with @mcollina , and I would change as well the test that you've linked...
Currently this is different from the example provided here https://github.com/delvedor/find-my-way/pull/34#issuecomment-335070418.

In the current example `/example/shared/nested/oops` is not defined in the router, so there's no much value in checking that we reach the default route.
When we register also an handler for the route `/example/shared/nested/oops`, we still expect to reach the  default route, because the initial part of tha parametric route is shadowed by the static `/example/shared/nested/test` route.
I guess this was also Matteo's point.